Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -3710,11 +3710,11 @@ def gen2021HiMix(fragment,howMuch):
 defaultDataSets['2017Design']='CMSSW_12_0_0_pre4-113X_mc2017_design_v5-v'
 defaultDataSets['2018']='CMSSW_12_0_0_pre4-113X_upgrade2018_realistic_v5-v'
 defaultDataSets['2018Design']='CMSSW_12_0_0_pre4-113X_upgrade2018_design_v5-v'
-defaultDataSets['2021']='CMSSW_12_0_0_pre4-120X_mcRun3_2021_realistic_v2-v'
+defaultDataSets['2021']='CMSSW_12_5_0_pre4-124X_mcRun3_2022_realistic_v10-v'
```

Comment:
Hi @kskovpen let me understand. 
Are these samples done using the vtx smearing as of https://github.com/cms-sw/cmssw/pull/38760

Reply:
They use Run3RoundOptics25ns13TeVLowSigmaZ

Reply:
I am confused then. 
What's the point of updating them?

Reply:
Shall we use Realistic25ns13p6TeVEarly2022Collision?

Reply:
yes, `Run3RoundOptics25ns13TeVLowSigmaZ` is the current vtx smearing while `Realistic25ns13p6TeVEarly2022Collision` will be the new one.

Reply:
Thanks, Marco